media/css/firefox/whatsnew/whatsnew-67.0.5.scss

```diff
@@ -0,0 +1,210 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/hero';
+@import '../../protocol/components/state-fxa';
+
+body {
+    background: url('/media/img/firefox/whatsnew_67/noodles.png') no-repeat;
+    background-size: 850px;
+    background-position-x: -350px;
+    background-position-y: $layout-md;
+}
+
+main {
+    color: $color-off-black;
+}
+
+//* -------------------------------------------------------------------------- */
+// Up-to-date page header
+
+.c-page-header {
+    background: #fff;
+    margin-bottom: $spacing-2xl;
+    margin-left: 0;
+}
+
+.c-page-header-up-to-date {
+    @include text-display-xxs;
+    display: none;
+    margin-top: $layout-xs;
+
+    span {
+        background-color: #6EE7C4;
+        border-radius: 2px;
+        display: inline-block;
+        font-weight: normal;
+        padding: $spacing-sm $spacing-md;
+        max-width: $content-sm;
+    }
+
+    .is-up-to-date & {
+        display: block;
+    }
+}
+
+@media #{$mq-md} {
+    .c-page-header-inner {
+        align-items: center;
+        display: grid;
+        grid-template-columns: 1fr max-content 1fr;
+    }
+
+    .c-page-header-up-to-date {
+        margin: 0 $layout-sm;
+    }
+}
+
+//* -------------------------------------------------------------------------- */
+// CTA
+
+.card {
+    background: #fff;
+    border-radius: 3px;
+    box-shadow: 1px 1px 10px 1px rgba(0, 0, 0, 0.21);
+    padding: $spacing-xl $spacing-2xl $spacing-xl $spacing-2xl;
+    margin: 0 auto;
+
+    h1 {
+        font-size: 2.25rem;
+        margin-bottom: 1em;
+    }
+
+    .show-fxa-supported-signed-in {
+        #monitor-logo {
+            float: left;
+        }
+
+        .mzp-l-hero-content {
+            margin-left: $layout-xl;
+        }
+    }
+
+    @media #{$mq-lg} {
+        width: 600px;
+    }
+
+    h2 {
+        @include text-display-sm;
+    }
+}
+
+//* -------------------------------------------------------------------------- */
+// Benefits
+
+.benefits {
```

Comment:
It would be nice to use the `c-` prefix for this component, but it's not a blocker.